dev/dc.otel.yml

```diff
@@ -0,0 +1,22 @@
+version: "3"
```

Comment:
You also need to overwrite `OPENTELEMETRY_ENABLED` for backend, reader and writer here so that it works in the dev setup, or do you do this somewhere else?

Reply:
I would propose not to use `OPENTELEMETRY_ENABLED` at all.  To disable OpenTelementry, you can set the sammpler to off with the default varialbes. If the sampler is off, then all opentelementry functions are noops

Reply:
also good for me, but I would still argue to enable it in the docker compose file, because there is not really a "manual" way to activate it for us developers in the dev setup.

Reply:
@ostcar @jsangmeister 
while I agree that we should use the default `OTEL_*` env vars for configuring e.g. the the endpoint of the exporter if needed. I think having our own var for turning it off alltogether can be more assuring for us, that there will no logging of any potentially sensitive data in production.
The introduction of the [env var specification](https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/) reads

> The goal of this specification is to unify the environment variable names between different OpenTelemetry SDK implementations. SDKs MAY choose to allow configuration via the environment variables in this specification, but are not required to. If they do, they SHOULD use the names listed in this document.

Which doesn't make me comfortable in relying on every SDK implementation we might use in the future to yield the same behavior for the the same env var values passed. It would require careful testing everytime.